backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/searchinputsummary/BpkSearchInputSummary.kt

```diff
@@ -45,15 +52,18 @@ fun BpkSearchInputSummary(
     onInputChanged: (String) -> Unit,
     clearAction: BpkClearAction,
     modifier: Modifier = Modifier,
+    type: BpkSearchInputSummaryType = BpkSearchInputSummaryType.TextInput,
 ) {
     BpkTextFieldImpl(
         value = inputText,
         onValueChange = onInputChanged,
         modifier = modifier,
+        readOnly = type is BpkSearchInputSummaryType.ReadOnly,
         placeholder = inputHint,
         prefix = prefix,
         status = BpkFieldStatus.Default,
         clearAction = clearAction,
         type = BpkTextFieldType.Search,
+        isFocused = type is BpkSearchInputSummaryType.ReadOnly && type.isFocused,
```

Comment:
small thing here: I think it'd be better to change this condition to only set a non-null value for the `ReadOnly` type as the default input one should use the system focused state